plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -230,6 +230,18 @@ public function legacy_rest_api_is_available() {
 		return class_exists( 'WC_Legacy_REST_API_Plugin', false );
 	}
 
+	/**
+	 * Get the WooCommerce version.
+	 *
+	 * @since 10.2.0
+	 *
+	 * @param bool $include_prerelease_info Whether to include pre-release information in the version string.
+	 * @return string The WooCommerce version.
+	 */
+	public function version( ?bool $include_prerelease_info = false ): string {
```

Comment:
As I go forward with the review, the naming here looks more confusing.

How about renaming it to prompt it to normalize the version and removing the parameter (so we keep referencing the property for the unprocessed version and the method where we need normalization)?

Reply:
Makes sense. I think I was trying to go the way of "soft" deprecating `->version` and have it replaced by `->version()` so that it worked for both prereleases and stables, obviously defaulting to no normalization (as `->version`) but you make a good point here.

Do you think the method needs to be renamed as well? Otherwise, with both `->version` and `->version()` I don't think things will be better than right now. Do you have any suggestions?